client.go

```diff
@@ -276,15 +277,28 @@ func (c *Client) UpdateEnvironment(ctx context.Context) error {
 		return errors.New(e["detail"])
 	}
 	c.environment.Store(&env)
+	ids := make(map[string]identities.IdentityModel)
```

Comment:
I don't love the variable naming here. I think by `ids` we're referring to identities, but at first glance, I was expecting just a numeric id for example.

Reply:
Changed the naming.